osbs/conf.py

```diff
@@ -113,13 +113,7 @@ def get_openshift_required_version(self):
                                     "openshift_required_version",
                                     can_miss=True)
         if verstring:
-            try:
-                openshift_required_version = [int(x)
-                                              for x in verstring.split('.')]
-            except ValueError:
-                pass
-
-            return openshift_required_version
+            return [int(x) for x in verstring.split('.')]
```

Comment:
Then it will fail with ValueError, whereas previously it would have failed with NameError.

Maybe it should use `pkg_resources.parse_version()` instead.